src/lib/utils.ts

```diff
@@ -33,6 +34,78 @@ export const getNodeTextContent = (node: ManuscriptNode) => {
   return textContent
 }
 
+export const getContributorTextContent = (
```

Comment:
Since we create all these functions for a similar purpose and they do same things, I suggest, we organise them accordingly: they can be moved into a common for them file or directory. Utils.ts is really more for assorted items that are shared between different components. 

Another neat thing that can be done here is to refactor those get{node}TextContent function in such a way that content getters are accessible by schema.nodeType.

Perhaps we need to consider moving those textContentGetters into transform since the can be directly associated with each nodeSchema.

Reply:
> Since we create all these functions for a similar purpose and they do same things, I suggest, we organise them accordingly: they can be moved into a common for them file or directory. Utils.ts is really more for assorted items that are shared between different components.
> 
sure, will do.

Reply:
> Another neat thing that can be done here is to refactor those get{node}TextContent function in such a way that content getters are accessible by schema.nodeType.
> 


are you suggesting to refactor the textContent getter, to return the whole content for the target node and its children nodes?

> Perhaps we need to consider moving those textContentGetters into transform since the can be directly associated with each nodeSchema.

Something like have a function inside the schema to return the text content of the node?

Reply:
What I meant is something like a function that would get text for a node type that can be displayed in the inspector. So instead of creating a bunch of functions like getTextContentFor{some}Node, you can call getDisplayTextFor<NodeTypes.Affiliation>(this.node) (this is just an example)

Reply:
So this is not a refactoring but a create of a function that would return a displayable text for the inspector

Reply:
done